src/components/molecules/Visualizer/WidgetAlignSystem/Area.tsx

```diff
@@ -57,6 +57,7 @@ export default function Area({
       align={(area === "middle" || section === "center") && widgets?.length ? align : undefined}
       style={{ flexWrap: wrapContent ? "wrap" : undefined }}
       editorStyle={{
+        flexWrap: wrapContent ? "wrap" : undefined,
```

Comment:
I prefer you put:
`flexWrap: wrapContent ? "wrap" : "nowrap",`
As it is clearer, and it is the flexWrap default value